### ISSUE TITLE
Ignore front matter configuration

### ISSUE THREAD

**Issue by twsh** [markdownlint/markdownlint#260]
I tried putting this in my .mdlrc:

`ignore-front-matter true`

I got errors the error:

```
syntax error, unexpected keyword_true, expecting keyword_do or '{' or '(' (SyntaxError)
ignore-front-matter true
```

I think that it's a typo in the documentation, and that it should be:

`ignore_front_matter true`

**Comment by lynn9388** [comment]
This issue has been mentioned many times #241 #170 #254 #185 #249 #181 .
@psyomn Could you merge the pull request from #170 or #241 to avoid unnecessary trouble?

**Comment by jaymzh** [comment]
Fix has been merged, other PRs closed.

**Comment by jaymzh** [comment]
This has been fixed in 0.6.0
